Dockerfile

```
@@ -0,0 +1,6 @@
+FROM openjdk:21-jdk-slim
+
+ARG JAR_FILE=build/libs/*.jar
+COPY ${JAR_FILE} app.jar
```

Hadolint (2.14.0)

[warning] `COPY` to a relative destination without `WORKDIR` set.

(DL3045)